### PR TITLE
Fixed for PGI and Intel CI builds

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

File: host-configs/toss_3_x86_64_ib/icpc_19_0_4_gcc8headers.cmake [hunk 1]
```diff
@@ -0,0 +1,10 @@
+##############################################################################
+# Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+# project contributors. See the COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (MIT)
+##############################################################################
+set(COMMON_FLAGS "-gxx-name=/usr/tce/packages/gcc/gcc-7.1.0/bin/g++ -std=c++11")
```
Comment (adrienbernede) [inline host-configs/toss_3_x86_64_ib/icpc_19_0_4_gcc8headers.cmake]: icpc_19_0_4_gcc8headers.cmake suggests that gcc 8 headers are used, which is the case in the toolchain host-config file. Overriding this here is not the intended way to do things. A new toolchain should be created:
icpc_19_0_4_gcc7headers.cmake.

File: .gitlab/ci/build_quartz.yml [hunk 2]
```diff
@@ -178,11 +178,12 @@ test_quartz_icpc_17_0_2_advanced:
...
-  variables:
-    COMPILER: "icpc_18_0_0"
-  extends: .test_quartz
-  needs: ["build_quartz_icpc_18_0_0"]
+# No more Intel 18
+# test_quartz_icpc_18_0_0:
+#   variables:
+#     COMPILER: "icpc_18_0_0"
+#   extends: .test_quartz
+#   needs: ["build_quartz_icpc_18_0_0"]
```
Comment (adrienbernede) [inline .gitlab/ci/build_quartz.yml]: Feel free to remove.

File: host-configs/toss_3_x86_64_ib/icpc_19_0_4_gcc8headers.cmake (hunk 1 above)
Comment (davidbeckingsale) [reply host-configs/toss_3_x86_64_ib/icpc_19_0_4_gcc8headers.cmake]: I was matching the naming of the configuration in radiuss-ci, but you are right - this is a stupid name.

In the radiuss-ci host config, common flags are defined, but not added anywhere: https://github.com/LLNL/radiuss-ci/blob/master/gitlab/conf/host-configs/toss_3_x86_64_ib/icpc_19_0_4_gcc8headers.cmake

**Comment by davidbeckingsale** [comment]
LGTM

**Comment by adrienbernede** [comment]
@davidbeckingsale 
Intel 18_0_0 build failed in CI, but used to work. The "test" job was commented but not the "build" job.

**Comment by davidbeckingsale** [comment]
I will make another PR and delete that build job. Intel 18 doesn't work after updating BLT to v0.3.0

**Pull request merged**